package.json

```diff
@@ -76,7 +76,7 @@
     "@veupathdb/prettier-config": "^1.0.0",
     "@veupathdb/react-scripts": "^1.0.1",
     "@veupathdb/tsconfig": "^1.0.1",
-    "@veupathdb/wdk-client": "^0.5.5",
+    "@veupathdb/wdk-client": "^0.5.10",
```

Comment:
You'll also want to update the `@veupathdb/wdk-client` `peerDependency` to `>=0.5.10`.